File: src/2003/4021277130/main.cpp
#include<iostream>
using namespace std;
int main()
{
    int temp=0,n=0,x,y=0,i=1;
    cin>>x;
    while(y==0)
    {
        n=0;
        temp=temp+i;
        for(int j=1;j<=temp;j++)
        {
            if(temp%j==0)
            {
                n++;
            }
        }
        if(x==n)
        {
            cout<<temp;
            y++;
        }
        i++;
    }
}

File: src/2003/4021277130/main.py
x=int(input("Enter The Num : "))
y=0
i=1
temp=0
while y==0 :
    n=0
    temp=temp+i
    j=1
    while j<=temp:
        if temp%j==0:
            n+=1
        j+=1 
    if x==n:
        print(temp)
        y+=1
    i+=1